examples/fluent-bit/amazon-opensearch-serverless/permissions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "es:*",
```

Comment:
if we could get opensearch team to help narrow down these perms that'd be awesome

Reply:
Narrowed down to nothing.